src/components/common/Header.js

```diff
@@ -64,17 +64,23 @@ function MainHeader(props) {
   return (
     <Layout style={{ ...scrollStyles, top: showHeader ? '0' : '-115px' }}>
       <Header className="header_div">
-        <img src={logo2} className="header_img" alt="HRF logo" />
-        <Search
-          className="search_bar"
-          placeholder="Search"
-          onSearch={onSearch}
-        />
-        <Button onClick={listView}>List</Button>
-        <Button onClick={thumbnailView}>Thumbnail</Button>
-        <Link to="/">
-          <Button type="default">Bookmarks</Button>
-        </Link>
+        {props.page === 'bar' ? (
```

Comment:
we should destructure this given that we are doing it with `searchDocs` early on